.gitignore

```diff
@@ -64,4 +64,5 @@ fastlane/report.xml
 fastlane/screenshots
```

Comment:
might be worth cleaning this up while we're in here, some of our gitignores have a huge iOS one, I've been trying to clean up while in there